pkg/host/internal/bridge/ovs/ovs_test.go

```diff
@@ -137,12 +137,26 @@ func createInitialDBContent(ctx context.Context, c client.Client, expectedState
 func validateDBConfig(dbContent *testDBEntries, conf *sriovnetworkv1.OVSConfigExt) {
 	Expect(dbContent.OpenVSwitch).To(HaveLen(1))
 	Expect(dbContent.Bridge).To(HaveLen(1))
-	Expect(dbContent.Interface).To(HaveLen(1))
-	Expect(dbContent.Port).To(HaveLen(1))
+	Expect(dbContent.Interface).To(HaveLen(2))
+	Expect(dbContent.Port).To(HaveLen(2))
 	ovs := dbContent.OpenVSwitch[0]
 	br := dbContent.Bridge[0]
-	port := dbContent.Port[0]
-	iface := dbContent.Interface[0]
+	var internalPort, port *PortEntry
```

Comment:
nit: i would create a port and interface map keyed by name.
that way if we have more later, there would be no need to do additional if conditions as below.

Reply:
Done